### ISSUE TITLE
AdminLTE page warning message on 2.5.x

### ISSUE THREAD

**Issue by debiansid** [pi-hole/AdminLTE#450]
**In raising this issue, I confirm the following (please check boxes, eg [X] - no spaces) Failure to fill the template will close your issue:**

- [] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [] The issue I am reporting can be *replicated*
- [] The issue I'm reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:**

2

---
**[FEATURE REQUEST | QUESTION | OTHER]:**

**[BUG | ISSUE] Expected Behaviour:**
I have installed pi-hole on lede-x86(which is variation of openwrt, so `pihole -d` would not be helpful.
basically,I just change location of pihole.log; pihole script base; hostname in auth.php.

pihole-AdminLTE works before 2.5.x, after 2.5.x , it always show that warning on the top of every page.

would you please tell me how to fix it?

```
Warning: session_start(): Cannot send session cache limiter - headers already sent (output started at /mnt/backup/www/html/admin-new/index.php:7) in /mnt/backup/www/html/admin-new/scripts/pi-hole/php/password.php on line 10
```
**[BUG | ISSUE] Actual Behaviour:**

**[BUG | ISSUE] Steps to reproduce:**

-
-
-
-

**(Optional) Debug token generated by `pihole -d`:**

`<token>`

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

**Comment by Mcat12** [comment]
Since you changed a lot of things in the scripts, it is probably due to something you changed incorrectly. You should go back and check your changes, to make sure it's actually a bug in Pi-hole and not something you misconfigured. A debug token would still be useful if you can't figure it out.

**Comment by DL6ER** [comment]
Okay, so let's look at this again in more detail:

> `output started at /mnt/backup/www/html/admin-new/index.php:7`

There is nothing that could output anything on this line neither the current `master` nor `devel` branch code, so please provide more details of what you modified where.

**Comment by debiansid** [comment]
Here is changes I made.
https://github.com/debiansid/AdminLTE
https://github.com/debiansid/pi-hole-for-lede

**Comment by joepv** [comment]
I have the same issue. I'm running piHole on openSUSE with a manual install and everything works fine exept the `Warning: session_start(): Cannot send session cache limiter - headers already sent (output started at /mnt/backup/www/html/admin-new/index.php:7) in /mnt/backup/www/html/admin-new/scripts/pi-hole/php/password.php on line 10` warning when opening the admin panel.

I did a manual update with the command `git pull origin master` in the `/var/www/html/admin` folder. Nothing special, worked with previous updates.

`pihole -d` tells me everything is ok.

**Comment by Mcat12** [comment]
Does this warning appear in the log or on the site itself? It doesn't appear to have a negative effect.

**Comment by joepv** [comment]
It appears on the website. Above all content. It's a PHP error.
It has no negative effect on the site I think, but I don't use a password.

**Comment by DL6ER** [comment]
Usually PHP is configured such that error are not shown on the pages but only in the logs (after all they can contain source code and hence can be a possible security risk).

I think you should disable them, but we will also make sure that they won't appear in future releases.

**Comment by Mcat12** [comment]
Does it appear as a browser alert like this, or in the page itself?
![image](https://cloud.githubusercontent.com/assets/4417660/24567452/c8850546-162b-11e7-804e-dcda05c7a9bc.png)

**Comment by debiansid** [comment]
It's in the page

**Comment by Mcat12** [comment]
Please provide a screenshot.

**Comment by debiansid** [comment]
![img_5683](https://cloud.githubusercontent.com/assets/9624961/24583356/142a7e6e-177a-11e7-90df-c0ba64c40aef.PNG)
